Gems/EMotionFX/Code/EMotionFX/Pipeline/RCExt/Motion/MotionDataBuilder.cpp

```diff
@@ -375,9 +376,10 @@ namespace EMotionFX
 
                 // Set the bind pose transform.
                 SceneAPIMatrixType bindBoneTransformNoScale(bindSpaceLocalTransform);
-                const AZ::Vector3    bindPos   = coordSysConverter.ConvertVector3(bindSpaceLocalTransform.GetTranslation());
                 const AZ::Vector3    bindScale = coordSysConverter.ConvertScale(bindBoneTransformNoScale.ExtractScale());
```

Comment:
Why are we not taking the scale from the converted bone transform here?

Reply:
bindBoneTransformNoScale.ExtractScale() took away the scale here. The idea is that the scale remain intact during any conversions, so we extract the scale first, do the conversion, then re-apply the scale back.